README.md

```diff
@@ -52,121 +52,6 @@ Alternatives: https://github.com/andymass/vim-matchup added tree-sitter support
 
 ## Supported languages
 
-<!--textobjectinfo-->
```

Comment:
It's actually triggered by this comment

Reply:
I also renamed `textobjectinfo` to `pairsinfo`